### PR TITLE
reintroduce binary compatabilty lost in #303

### PR DESCRIPTION
**Pull request by jtnord**
alternative to #307 proposed by @jglick 

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java [hunk 1]
```diff
@@ -475,15 +476,21 @@ public HttpResponse doTerm() {
             @Override public void onFailure(@NonNull Throwable t) {}
         });
         printLater(StopState.KILL, "Click here to forcibly kill entire build");
+    }
+
+    /** Sends {@link StepContext#onFailure} to all running (leaf) steps. */
+    @RequirePOST
+    @WebMethod(name = { "term" })
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java:483]:
```suggestion
    @WebMethod(name = "term")
```
conventionally

Comment (jglick) [reply src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java:483]: probably also want to mark `DoNotUse` to make it clear this is here only for use from Stapler

**Review by jglick — APPROVED** [review]
Does it work?

**Comment by jtnord** [comment]
> Does it work?

yes

**Review by jglick — APPROVED** [review]

**Pull request merged**